ui-react/src/atoms/progressCounterItem/component.tsx

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import styled from 'styled-components';
 import { getClassName } from '@kibalabs/core';
 
-import { IComponentProps, defaultComponentProps, themeToCss, useBuiltTheme } from '../..';
+import { useBuiltTheme } from '../../theming';
```

Comment:
updated as i copied-pasted this component to create the TabBarItem component and found these needed fixes